Include <unistd.h> for the read and write functions, to avoid implicit
function declarations and build failures with future compilers.

diff --git a/cmain.c b/cmain.c
index ec0558d11cdd325d..aced8300d55d17fd 100644
--- a/cmain.c
+++ b/cmain.c
@@ -19,6 +19,8 @@
 #include "config.h"
 #endif
 
+#include <unistd.h>
+
 #include "compface.h"
 
 /* the buffer is longer than needed to handle sparse input formats */
diff --git a/uncmain.c b/uncmain.c
index a94279df1ff73223..e52831d0c0f117e1 100644
--- a/uncmain.c
+++ b/uncmain.c
@@ -19,6 +19,8 @@
 #include "config.h"
 #endif
 
+#include <unistd.h>
+
 #include "compface.h"
 
 /* the buffer is longer than needed to handle sparse input formats */
